analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/ObjectsShouldNotBeDisposedMoreThanOnceBase.cs

```diff
@@ -28,7 +28,7 @@ public abstract class ObjectsShouldNotBeDisposedMoreThanOnceBase : SymbolicRuleC
     protected const string MessageFormat = "Resource '{0}' has already been disposed explicitly or implicitly through a using statement. Please remove the redundant disposal.";
     protected abstract ILanguageFacade Language { get; }
 
-    private static readonly string[] DisposeMethods = { "Dispose", "DisposeAsync"};
+    protected static readonly string[] DisposeMethods = { "Dispose", "DisposeAsync"};
```

Comment:
This will be moved in the implementation of C# during the VB rule PR.

Reply:
I'm not sure about that. It's needed to work with the symbol. So it will just not be used by the VB